wire-ios/Wire-iOS/Sources/UserInterface/Settings/AccentColorPicker.swift

```diff
@@ -0,0 +1,81 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import SwiftUI
+import WireCommonComponents
+import WireSyncEngine
+
+struct AccentColorPicker: View {
+
+    @State var selectedColor: AccentColor
```

Comment:
I think state should be private.

Reply:
I don't think it can be private since  @State properties drive the view’s state and must be mutable and accessible within the view’s body, thus cannot be private.